modules/stac/providers.tf

```diff
@@ -6,5 +6,9 @@ terraform {
     kubernetes = {
       source = "hashicorp/kubernetes"
     }
+    http = {
+      source = "hashicorp/http"
+      version = "3.4.0"
+    }
```

Comment:
I don't think we need the http provider anymore since we're running this from a Kubernetes Job.